### PR TITLE
cn-pr-trans-readme-cn

### PR DESCRIPTION
**Pull request by markthink**
Chinese translation team documentation.

### CODE REVIEW

**Comment by stewart-yu** [comment]
@markthink Thanks for your contribution, but seems no need in `website` repo
/cc @zacharysarah WDYT?

**Comment by zhangxiaoyu-zidif** [comment]
ah, this document could be in website repo, but shall we move it to some directory...

**Comment by markthink** [comment]
@chenopis @zhangxiaoyu-zidif Updated the Chinese homepage link, trouble to help merge, thank you..

**Comment by dankohn** [comment]
Cc @kbarnard10 This adds the links to the KubeCon China site.

**Comment by zhangxiaoyu-zidif** [comment]
gernerally  /lgtm
but I am not sure if the format and file path are proper.

**Comment by zacharysarah** [comment]
@markthink 👋 Please open this PR in the [Chinese translation site](https://github.com/kubernetes/kubernetes-docs-zh) repo instead. We want to avoid duplicating that team's work here. Thanks in advance!

**Comment by tengqm** [comment]
@zacharysarah There seem to be some misunderstandings here because apparently you don't read Chinese, :)

In the PR there is a paragraph (in Chinese) describing the situation. Let me translate it for you:

> The upstream `website` repo has successfully migrated from `Jeklly` to `Hugo` on July 9th, 2018,
> and it now provides good support to Chinese contents. The official `kubernetes.io` website can now
> easily switch to Chinese language. We have decided to stop maintaining the
>[Chinese Translation Repo](https://github.com/kubernetes/kubernetes-docs-zh).
> The repo will be kept frozen.
> Previous translations can still be found [here](https://k8smeetup.github.io).
> The contributions made by the translators can still be viewed [here](https://k8smeetup.github.io/chart/).

**Comment by zacharysarah** [comment]
@tengqm Thanks for the translation! 🙇 

> We have decided to stop maintaining the Chinese Translation Repo.

If true, this is completely new to me and would be an otherwise unannounced and undiscussed change. @hanjiayao PTAL

**Comment by zacharysarah** [comment]
/hold

🛑 DO NOT MERGE 🛑 

Do not merge this PR or remove the `/hold`.

File: README-CN.md [hunk 1]
```diff
@@ -0,0 +1,163 @@
+# K8SMeetup 翻译流程与翻译校稿规范
+
+time：2018-07-11 author：xiaolong@caicloud.io
+
+由于 `website` 上游仓库从在 2018年7月9日，从 `Jeklly` 成功迁移至 `hugo`，且提供了完善的中文化支持，目前官网 `kubernetes.io` 已经可以轻松切换中文站点。因此原有的 [中文翻译仓库](https://github.com/kubernetes/kubernetes-docs-zh) 不在维护，继续冻结中。往期的翻译内容仍然可以[这里](https://k8smeetup.github.io)访问，对应往期的翻译者贡献仍然可以[这里](https://k8smeetup.github.io/chart/) 查看。
```
Comment (hanjiayao) [inline README-CN.md]: i think this is misunderstood - from the email that Zach sent on June 20, there's a line saying "We changed the name of the Chinese translation repository to k/docs-kubernetes-zh. GitHub automatically aliases the new name, so contributors should experience no problems." If I understand correctly, it's still the same repo but with a different alias (name) now.

**Pull request closed without merging**